data/json/dreams.json

```diff
@@ -660,6 +714,25 @@
     "type": "dream",
     "messages": [
       "You curl up some more in your nest.  Geez, shredded paper really *is* comfy.",
+      "If you're going to feather a nest, you've got to learn about how to treat a lady.",
+      "You're stuck in a maze.  But you know there is cheese at the end - you can smell it!",
+      "Wolf, hawk, fox, and snake can't stand in my way.  My body is weak and it may break, but not today.",
+      "Oh boy! This hidey hole sure is swell.",
+      "Run, run, run, run, run, run, stop!  Smell⋯  Listen⋯  Run, run, run, run, stop!  Smell that?  Food!  Run, run, run, run.",
+      "Chew, gnaw, bite and chew.  We'll make quick work of these ropes and let loose the lion!",
```

Comment:
> We'll make quick work of these ropes and let loose the lion!",

wat?

Reply:
Aesop's fable demonstrating the power of the mouse - https://fablesofaesop.com/the-lion-and-the-mouse.html

Reply:
Did you still want to remove the aesop fable one about chewing through the ropes?